setup.py

```diff
@@ -45,12 +45,23 @@ def get_versions():
 
 includes.append(os.path.join(boost_prefix, 'include'))
 if 'BOOST_VERSION' in os.environ:
-    includes.append(os.path.join(boost_prefix, 'include',
-                                 "boost-%s" %os.environ['BOOST_VERSION']))
+    for boost_dir in ('boost-%s', 'boost%s'):
+        if os.path.isdir(os.path.join(boost_prefix, 'include', boost_dir % os.environ['BOOST_VERSION'])):
+            includes.append(os.path.join(boost_prefix, 'include', boost_dir %os.environ['BOOST_VERSION']))
+            break
+    else:
+        raise Exception("Cannot find boost headers")
+elif 'BOOST_PREFIX' in os.environ:
+    if os.path.isdir(os.path.join(boost_prefix, 'include', 'boost')):
+        includes.append(os.path.join(boost_prefix, 'include', 'boost'))
+    else:
+        raise Exception("Cannot find boost headers")
 
 if 'BOOST_VERSION' in os.environ:
     libdirs.append(os.path.join(boost_prefix, 'lib'))
```

Comment:
This was likely a typo and meant to be BOOST_PREFIX. I think it can go away then.